config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
   root "articles#index"
 
-  resources :articles
+  resources :articles do
+    resources :comments
```

Comment:
What is done on this line?

Reply:
This code creates a comment path as a nested resource within the article path, thus maintaining the hierarchical relationship of each.